src/modules/package.rs

```diff
@@ -102,9 +102,24 @@ fn get_setup_cfg_version(context: &Context, config: &PackageConfig) -> Option<St
 }
 
 fn get_gradle_version(context: &Context, config: &PackageConfig) -> Option<String> {
-    let file_contents = context.read_file_from_pwd("build.gradle")?;
-    let re = Regex::new(r#"(?m)^version ['"](?P<version>[^'"]+)['"]$"#).unwrap();
-    let caps = re.captures(&file_contents)?;
+    let build_file_contents = match context.read_file_from_pwd("build.gradle") {
+        Some(contents) => contents,
+        None => "none".to_string(),
```

Comment:
Is `"none"` used in `build.gradle`? If not, please stick with `Option<String>`, and if it does use `"none"` please filter `Some("none")` into `None` instead and add a test that uses `version "none"`.

Reply:
I feel like I should probably stick with Option anyway, since that seems to be the intended Rust way.